build-system/tasks/get-zindex/index.js

```diff
@@ -32,7 +33,7 @@ const tableOptions = {
   hsep: '   |   ',
 };
 
-const preamble = 'Run `gulp get-zindex` to generate this file.';
+const preamble = '**Run `gulp get-zindex --fix` to generate this file.**';
```

Comment:
@raxsha @processprocess @lannka @gmajoulet 

PSA for people who've modified [css/Z_INDEX.md](https://github.com/ampproject/amphtml/commits/master/css/Z_INDEX.md) in the past. The command to update said file now requires `--fix`: 

```
gulp get-zindex --fix
```